specs/UrbanMobility/ArrivalEstimation/doc/spec.md

```diff
@@ -0,0 +1,67 @@
+# ArrivalEstimation
+
+## Description
+
+This Entity Type captures the estimation of arrival of a public transport vehicle, performing service on a route, to a certain stop. 
+
+## Data Model
+
++ `id`: Entity Id
+  + It shall be `urn:ngsi-ld:gtfs:ArrivalEstimation:<identifier>`. 
+
++ `type`: Entity Type 
+  + It shall be equal to `ArrivalEstimation`
+  
++ `dateCreated` : Entity's creation timestamp.
+  + Attribute type: [DateTime](https://schema.org/DateTime)
+  + Read-Only. Automatically generated. 
+ 
++ `dateModified` : Last update timestamp of this Entity.
+  + Attribute type: [DateTime](https://schema.org/DateTime)
+  + Read-Only. Automatically generated.
+  
++ `hasStop` : Stop to which this estimation applies to.
```

Comment:
Since this is a relationship, I assume that this attribute should be called `refStop` - compare this with `WaterQualityObserved.refPointOfInterest`

Reply:
updated guidelines go in this PR

https://github.com/Fiware/dataModels/pull/265

Reply:
Model aligns with NGSI-LD - NTC